config/routes.rb

```diff
@@ -8,5 +8,11 @@
   get "/users/:id", to: "users#show"
 
   get "/users/:user_id/posts", to: "posts#index"
+  get "/users/:user_id/posts/new", to: "posts#new"
+  post "/users/:user_id/posts", to: "posts#create"
   get "/users/:user_id/posts/:id", to: "posts#show"
+
+  post "/users/:user_id/posts/:id/comments", to: "comments#create"
+
+  post "/users/:user_id/posts/:id/likes", to: 'likes#create'
```

Comment:
- [ ] Please make sure that you have a route `GET /posts/:post_id/comments/new(.:format) comments#new`  to get new comment in order to make your app works as expected :+1: